exercism/diamond/diamond.js

```diff
@@ -0,0 +1,15 @@
+var Diamond = function diamond(letter) {
```

Comment:
This function declaration is repeated. I would do this... `const diamond = function( letter ){`